external/settings.php

```diff
@@ -7,6 +7,12 @@
 
 $tmpl = new OCP\Template( 'external', 'settings');
 
-$tmpl->assign('images', glob(\OC_App::getAppPath('external') . '/img/*.*'));
+$images = glob(\OC_App::getAppPath('external') . '/img/*.*');
+$theme = \OC_Util::getTheme();
```

Comment:
@scolebrook you could use `\OC::$server->getSystemConfig()->getValue("theme", '');` for that.

Note that if no theme is set, the string is null. However the utility function would detect that and replace it with "default" under some conditions: https://github.com/owncloud/core/blob/master/lib/private/legacy/util.php#L1291

Not sure if that applies to your use case though.